embassy-rp/Cargo.toml

```diff
@@ -20,6 +20,8 @@ defmt = ["dep:defmt", "embassy-usb?/defmt"]
 # There are no plans to make this stable.
 unstable-pac = []
 
+time-driver = []
```

Comment:
Could you enable it in `package.metadata.embassy_docs.features`, so it shows up in docs?